main.go

```diff
@@ -204,7 +207,7 @@ func registerWorkloads(port string, log logger.Handler) {
 		log.Info(err.Error())
 		return
 	}
-	log.Info("Successfully registered latest service mesh components with Meshery Server at ", mesheryServerAddress())
+	log.Info("Successfully registered latest components for version ", version)
```

Comment:
```suggestion
	log.Info("Registered latest components for version ", version)
```

Do we know how many components? If so, can this information be included?